<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Write</string>
  </property>
  <widget class="QTextEdit" name="contentTextLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>711</width>
     <height>461</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Gulim'; font-size:8.76522pt;&quot;&gt;내용을 입력하세요&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="titleTextLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>64</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>제목</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="categoryComboBox">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>자유</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>취미</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>연애</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>고민</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>583</x>
     <y>60</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>카테고리</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>590</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>등록</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>590</y>
     <width>89</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="locInfoCheckBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>589</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>작성자의 현 위치가 사용될 수 있음을 인지했습니다.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
